### DISCUSSION TITLE
Beta Version 2.6.1 (99E12b4) bug?

### DISCUSSION THREAD

**Discussion by djbetterly** [artisan-roaster-scope/artisan#896] [General]
I'm running the beta version listed above, when my PID is active, regardless of what I have the SV at, it will not change from 359.  I'm assuming that's a bug?  Or am I mistaken..

I'll download the newest release today and give it a try.

**Comment by MAKOMO** [comment]
Your report is unclear to me. Please give step-by-step instructions on how to reproduce the effect that you see. It is also unclear to me what the wrong behavior is you report. What des not change from 359? The SV?
